lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -25,6 +25,8 @@ class CatalogController < ApplicationController
       'collection.rows': 1
     }
 
+    config.highlight_field = 'text'
```

Comment:
@jcoyne it may be obvious but could we put a comment that is like, this sets which indexed field will display with highlighted matches per search ?

Reply:
I think this is a great idea. Will do.